packages/gestalt/src/useTapFeedback.js

```diff
@@ -31,7 +34,21 @@ export default function useTapFeedback(): {|
     x: 0,
     y: 0,
   });
+
+  const [compressStyle, setCompressStyle] = React.useState(null);
+
+  React.useEffect(() => {
+    if (ref && ref.current) {
+      setCompressStyle({
+        transform: isTapping
```

Comment:
instead of dynamic value, can we just use "95%" instead? we used to use 95% for part of our codebase and it might be a nice simple default.

Reply:
@jackhsu978 the 95% doesn't work for bigger containers - they compress too much. It might work for mweb but not on desktop web.

Before
![link-before-static](https://user-images.githubusercontent.com/127199/87733592-1a0f2780-c785-11ea-9390-e5352afff294.gif)

After
![link-after-static](https://user-images.githubusercontent.com/127199/87733596-1b405480-c785-11ea-98e9-db4b40ca35dd.gif)

Reply:
got it. thanks!